### ISSUE TITLE
App constantly closing

### ISSUE THREAD

**Issue by AE102** [malensek/3RVX#108]
I have been using 3RVX for a few years now.  When I started up my computer I tried using the hotkeys that I had set up. Nothing happened. I checked my task manager and nothing came up. I went into my file explorer and tried to manually open it nothing happened. When I went into the task manager I saw the app flashing but then closed almost immediately.  Is there a way to fix it?
_Please Note: The setting app is still opening and I can change the settings.And I have also tried to reinstall it._

**Comment by Angrist17** [comment]
I have the exact same issue. Any luck with a solution?

**Comment by AE102** [comment]
@Angrist17 No, But I just downgraded to version 2.9.1 and it works fine. (https://3rvx.com/release-notes/2.9.1.html)
